Libraries/StyleSheet/StyleSheetTypes.d.ts

```diff
@@ -69,9 +69,15 @@ export interface FlexStyle {
     | undefined;
   left?: number | string | undefined;
   margin?: number | string | undefined;
+  marginBlock?: number | string | undefined;
```

Comment:
As we know, this type;
```ts
number | string | undefined
```
is a generic type for most of the style attributes. 
Should we declare a generic type for this type as well? Ex. `BoxDimensionValue` or `BoxDimensionType` etc.

And use the same generic type in below code. This will allow us to DRY the code.

Similarly for ;
```ts
number | undefined
```

What do you think?

Reply:
Contributions welcome to clean up the TS declarations 🙂.